### PR TITLE
Docs: document how to terminate a session

### PR DESCRIPTION
**Pull request by stsewd**
There is an open PR that implements this on .com. Wanted to link to this "feature" from all places where we show authn/authz.

### CODE REVIEW

File: docs/versions.rst
Comment (ericholscher) [inline docs/versions.rst:123]:
```suggestion
When you login to a documentation site, 
you will be logged in until close your browser.
If you want to force a log out,
you can do so by clicking on the :guilabel:`Logout` button that is on the flyout menu in the theme.
This is usually located in the bottom right or bottom left, 
depending on the theme design.
```

File: docs/commercial/sharing.rst
Comment (ericholscher) [inline docs/commercial/sharing.rst:39]: Having too many notes feels a bit heavy in the design. I think this should probably just be part of the content.

File: docs/commercial/single-sign-on.rst
Comment (ericholscher) [inline docs/commercial/single-sign-on.rst:20]:
```suggestion

 Users can log out by using the  :ref:`Log Out <versions:Logging out>`. button in the RTD flyout.

```

**Review by ericholscher — APPROVED** [review]
Looks good, once we expose the button to users.

File: docs/versions.rst
Comment (agjohnson) [inline docs/versions.rst:120]: Verb form log in here

```suggestion
When you log in to a documentation site, you will be logged in until close your browser.
```

File: docs/versions.rst
Comment (agjohnson) [inline docs/versions.rst:122]: :axe: Fewer words for more clarity :axe: 

```suggestion
To log out, click on the :guilabel:`Log out` link in your documentation's flyout menu.
```

File: docs/commercial/sharing.rst
Comment (agjohnson) [inline docs/commercial/sharing.rst:37]: We should eventually find a better way to describe the menu. I doubt folks know what we mean when core team says flyout.

For now, "link" is more clear, as we don't have any buttons in the flyout menu.

```suggestion
Users can log out by using the :ref:`Log Out <versions:Logging out>` link in the RTD flyout menu.
```

File: docs/commercial/single-sign-on.rst
Comment (agjohnson) [inline docs/commercial/single-sign-on.rst:17]:
```suggestion
Users can log out by using the :ref:`Log Out <versions:Logging out>` link in the RTD flyout menu.
```

**Review by agjohnson — COMMENTED** [review]
Looks good. Noted some language updates for additional clarity to the user. I'll block this PR on the underlying link PR.

File: docs/versions.rst
Comment (ericholscher) [inline docs/versions.rst:124]:
```suggestion
This will log you out from the current domain,
but not end any other session that you have active. 
```

File: docs/versions.rst
Comment (humitos) [inline docs/versions.rst:124]: Doesn't this log you out from the dashboard as well?

Comment (stsewd) [reply docs/versions.rst:124]: Nope, but we redirect to the dashboard after logging out from the domain

Comment (humitos) [reply docs/versions.rst:124]: Do we redirect to the Dashboard Homepage or the Dashboard Logout page?

Comment (stsewd) [reply docs/versions.rst:124]: I think we are still redirecting to the logout page, I think we should redirect to the home page TBH

Comment (ericholscher) [reply docs/versions.rst:124]: Agreed, that seems cleaner 👍

**Pull request merged**
